packages/material/scss/scheduler/_theme.scss

```diff
@@ -45,18 +42,37 @@ $scheduler-event-text: $primary-contrast !default;
         background-color: $scheduler-event-bg;
         color: $scheduler-event-text;
 
-        .k-event-actions .k-link {
-            color: $scheduler-event-bg;
-            background-color: $scheduler-event-text;
+        .k-event-template {
+            padding: $padding-y-sm $padding-x-sm;
+        }
+
+        .k-event-actions {
+            right: $padding-x-sm;
+
+            .k-link {
+                color: $scheduler-event-bg;
+                background-color: $scheduler-event-text;
+            }
         }
 
         .k-resize-handle::before {
             // TODO: $scheduler-content-area-bg
             background-color: $primary-contrast;
         }
-    }
-    .k-event.k-state-selected {
-        @include appearance( selected-node );
+
+        &.k-state-selected {
+            @include appearance( selected-node );
+            background-color: $scheduler-event-bg;
```

Comment:
Removing both lines(`@include app...` and `backgorund-col...`) will result in the same.